### ISSUE TITLE
upgrade to elasticsearch 6

### ISSUE THREAD

**Issue by alee** [comses/comses.net#357]
ES6 now supported in wagtail backends:

http://docs.wagtail.io/en/v2.1/topics/search/backends.html#wagtailsearch-backends-elasticsearch

**Comment by alee** [comment]
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.5/dist-packages/wagtail/search/management/commands/update_index.py", line 125, in handle
    schema_only=options.get('schema_only', False), chunk_size=options.get('chunk_size')
  File "/usr/local/lib/python3.5/dist-packages/wagtail/search/management/commands/update_index.py", line 77, in update_backend
    index.add_model(model)
  File "/usr/local/lib/python3.5/dist-packages/wagtail/search/backends/elasticsearch2.py", line 839, in add_model
    update_all_types=True
  File "/usr/local/lib/python3.5/dist-packages/elasticsearch/client/utils.py", line 76, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/elasticsearch/client/indices.py", line 271, in put_mapping
    '_mapping', doc_type), params=params, body=body)
  File "/usr/local/lib/python3.5/dist-packages/elasticsearch/transport.py", line 318, in perform_request
    status, headers_response, data = connection.perform_request(method, url, params, body, headers=headers, ignore=ignore, timeout=timeout)
  File "/usr/local/lib/python3.5/dist-packages/elasticsearch/connection/http_urllib3.py", line 185, in perform_request
    self._raise_error(response.status, raw_data)
  File "/usr/local/lib/python3.5/dist-packages/elasticsearch/connection/base.py", line 125, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
elasticsearch.exceptions.RequestError: TransportError(400, 'mapper_parsing_exception', 'Mapping definition for [faq_entry_id] has unsupported parameters:  [analyzer : edgengram_analyzer] [search_analyzer : standard]')
```

**Comment by alee** [comment]
fixed ES mapping error in 7aa368039aa3e04617d8bcab05b42273c879223c - 

Still need to look into how to properly index the individual FaqEntrys in the FAQ Page - perhaps skip the middleman FaqEntryPlacement and just have FaqEntry referencing the FaqPage
